Cubical/Foundations/Equiv/Properties.agda

```diff
@@ -191,3 +191,13 @@ compr≡Equiv p q r = congEquiv ((λ s → s ∙ r) , compPathr-isEquiv r)
 -- composition on the left induces an equivalence of path types
 compl≡Equiv : {A : Type ℓ} {a b c : A} (p : a ≡ b) (q r : b ≡ c) → (q ≡ r) ≃ (p ∙ q ≡ p ∙ r)
 compl≡Equiv p q r = congEquiv ((λ s → p ∙ s) , (compPathl-isEquiv p))
+
+isEquivFromIsContr : {A : Type ℓ} {B : Type ℓ′}
+                   → (f : A → B) → isContr A → isContr B
+                   → isEquiv f
+isEquivFromIsContr f isContrA isContrB =
```

Comment:
Hmm, can't one prove that this is an equivalence directly? I feel like it might be easier, but I might be wrong in which case this proof is fine...

Reply:
I made a direct proof. It avoids isoToEquiv, but it doesn't look better. But that might be due to me avoiding cubical primitives...

Reply:
Note there is also a `isContr→Equiv` in `Foundations.Equiv`.

Reply:
Yes, but I thought that might be a bit awkward to use to show, that some particular function is an equivalence.

Reply:
But I guess my proof would look nicer, if I use that.

Reply:
Ok, it does look better.

Reply:
Yeah, that looks good!